.github/workflows/prisma.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - "prisma/**"
   push:
     tags:
-      - v*
+      - prisma/v*
```

Comment:
@alexolivier just flagging this in case it somehow breaks your prisma publish flow?